test/common/dns.js

```diff
@@ -0,0 +1,290 @@
+'use strict';
+
+// Naïve DNS parser/serializer.
+
+require('../common');
```

Comment:
You could  replace it with `/* eslint-disable required-modules */` at the top of the file the way it is in `test/common/wpt.js`. I think I'd prefer that. If it's leaking globals, then any tests that use it will complain, so we're covered without importing the common module.`